src/main/kotlin/dev/arbjerg/lavalink/client/loadbalancing/regionFiltering.kt

```diff
@@ -0,0 +1,19 @@
+package dev.arbjerg.lavalink.client.loadbalancing
+
+import dev.arbjerg.lavalink.client.LavalinkNode
+
+interface RegionFilter {
```

Comment:
I'd prefer interfaces to be prefixed with an uppercase `I`
```suggestion
interface IRegionFilter {
```

Reply:
Perhaps you could browse through this SO question and tell me if you still have the same opinion? Otherwise I'll just commit this change.

https://softwareengineering.stackexchange.com/questions/117348/should-interface-names-begin-with-an-i-prefix

Reply:
I see where you are coming from, but personally I would like to easily distinguish between interfaces and abstract classes that is why I always follow the convention to prefix interfaces with `I` and abstract classes with `Abstract`.